### DISCUSSION TITLE
custom firmware?

### DISCUSSION THREAD

**Discussion by molebdynum** [roleoroleo/yi-hack-MStar#342] [Q&A]
I copied the custom firmware files into the micro sd card, but the camera does not self install them. I don't see the yellow light blinking.

**Comment by MD-V** [comment]
Model?
Firmware version?

**Reply by molebdynum** [reply]
9FUS 
y203c_0.4.0

**Reply by MD-V** [reply]
How does your SD card folder/file structure look like?

**Comment by molebdynum** [comment]
sys_y203c
home_y203c
folder-record
folder-log

**Comment by roleoroleo** [comment]
What's the fw version you read in the app?

**Reply by molebdynum** [reply]
8.2.0.0A_201912270941

**Reply by roleoroleo** [reply]
https://github.com/roleoroleo/yi-hack-Allwinner

**Reply by molebdynum** [reply]
roleoroleo.. That worked. Thank you very much!
